src/components/AccessibilityGuide/AriaLandmarks.mdx

```diff
@@ -0,0 +1,125 @@
+
+The banner landmark is an area that has global, rather than page-specific, information or actionable elements.
+For example, a website's logo and primary navigation would be expected child elements of a `banner` landmark. There should only be a single `banner` landmark per interface.
+
+The Header is Reservoir's only component that falls under the banner category. Our `Header` contains other landmarks: `complementary` for temporary sitewide alerts, `navigation` for the site navigation.
+
+Note: The DS `Banner` is a `complementary` landmark, despite its title matching `banner`.
+
+### complementary
+
+The `complementary` landmark indicates an area that contains additional information that "complements" the main content. Though complementary, the content it represents should also be understandable on its own.
+
+The `Banner`, `StatusBadge`, `Notification` , `Hero`, and `FeaturedContent` components could all be `complementary` landmarks.
+
+### contentinfo
+
+`contentinfo` is another globally repeating landmark, like `banner`, with information about the parent document. If `banner` is usually the site's header, `contentinfo` is usually the footer. There should only be a single `contentinfo` landmark per interface.
+
+The DS `Footer` uses the `contentinfo` role.
+
+### form
+
+The `form` landmark identifies a region that contains a collection of elements that combine to create a form: usually `button`, `input`,
+`select`, and `textarea`.
+
+<Banner
+  content={
+    <>
+      <strong>IMPORTANT:</strong> Use the search landmark instead of the form
+      landmark when the form is used for search functionality.
+    </>
+  }
+  type="warning"
+/>
+
+Interfaces can have multiple `form` regions, and each should be identified with a unique label.
+
+Elements under [Form Elements](../?path=/docs/components-form-elements) should be wrapped by the `form` landmark.
+
+### main
+
+The `main` landmark is meant to designate the primary content of a page. In most browsers, it's natively conveyed by use of the `<main>` HTML element.
+
+There should only be a single `main` landmark per interface. Since `main` represents the primary content of a page, it is the most important destination for [skip links](../?path=/docs/accessibility-guide-skip-navigation--docs), bypassing recurring elements like the header or navigation.
+
+The `Template` [component](../?path=/docs/components-page-layout-template--docs) renders the `main` landmark (and the `header` and `footer`).
+
+### navigation
+
+The `navigation` landmark identifies groups (e.g. lists) of links that are intended to be used for website or page content navigation.
+There can be multiple `navigation`s on a page, but if a `navigation` landmark has the same set of links as another `navigation` landmark on the page (think mobile versus desktop), use the same HTML `label` attribute for both.
+
+Elements under [Navigation](../?path=/docs/components-navigation) should be wrapped by the `navigation` landmark.
```

Comment:
Only two components in our `Navigation` category do not have a `<nav>` element baked in. In fact, one of the two that doe snot have it should be updated to have it added. So that would leave `Link` as the only component that needs to potentially be wrapped in a `navigation` landmark, and even that would depend on how it was being used.

This is a long winded way of saying this comment should be reworded.